### PR TITLE
Change plural sentence to singular

### PR DESCRIPTION
**Pull request by peterhauke**

### CODE REVIEW

File: content/docs/reconciliation.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ This will destroy the old `Counter` and remount a new one.
 
 >Note:
 >
->These methods are considered legacy and you should [avoid them](/blog/2018/03/27/update-on-async-rendering.html) in new code:
+>This method is considered legacy and you should [avoid it](/blog/2018/03/27/update-on-async-rendering.html) in new code:
```
Comment (harish-sethuraman) [inline content/docs/reconciliation.md:48]: There are two methods mentioned below 
- UNSAFE_componentWillUpdate()
- UNSAFE_componentWillReceiveProps()
And the already existing version seems to make sense.

Comment (peterhauke) [reply content/docs/reconciliation.md:48]: Yes, I saw the other note below. There is only one method listed in the first note, so the sentence does not read correctly when it is in the plural, but I agree, it does seem to make sense.

Maybe, the following would be more clear and better:

Note:
Methods prefixed with UNSAFE_ are considered to be legacy and you should avoid them in new code, for example:
* UNSAFE_componentWillMount()

and later

Note:
Methods prefixed with UNSAFE_ are considered to be legacy and you should avoid them in new code, for example:
* UNSAFE_componentWillUpdate()
* UNSAFE_componentWillReceiveProps()

If you agree, then I will make another PR with this proposal.

Comment (harish-sethuraman) [reply content/docs/reconciliation.md:48]: Hey really sorry. Now it makes sense to me I missed out the first note.

**Comment by harish-sethuraman** [comment]
Before:
Screenshot 2022-06-19 at 9 49 23 PM

After:
Screenshot 2022-06-19 at 9 49 06 PM

**Comment by peterhauke** [comment]
Thanks, @harish-sethuraman

I understand. It happens. I would go with the original PR then.

Do I need to make another PR or is this one OK now?

File: content/docs/reconciliation.md (hunk 1 above)
Comment (peterhauke) [reply content/docs/reconciliation.md:48]: Hello @harish-sethuraman , can you merge my original commit if it is OK with you?

**Comment by harish-sethuraman** [comment]
Thanks!

**Pull request merged**